.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,9 +2,9 @@ Checklist:
 
 <!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->
 
-- [ ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
-- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
-- [ ] My pull request targets the `main` branch of freeCodeCamp.
+- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
+- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
+- [x] My pull request targets the `main` branch of freeCodeCamp.
 
```

Comment:
Please undo these changes. This will change the template for everyone who uses it.

```suggestion
- [ ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ ] My pull request targets the `main` branch of freeCodeCamp.

```